### PR TITLE
Redesign the blog

### PR DESCRIPTION
**Pull request by joaoramos**
This pull request introduces a redesign of all pages of the blog. There are still a few things missing here, though:

- [x] Instructions on how to remove the Contentful greeting section on the top of the template.
- [x] Update greeting text and add design to the _Try the web app_ button.
- [x] Improve appearance of the copy within the hero section of the home page.
- [ ] Update the content in the space, in Contentful.
- [x] Improve appearance of the top bar for blog posts.
- [ ] Add message for empty collection (nice to have)

### CODE REVIEW

**Comment by martzoukos** [comment]
Some changes:
- [ ] The "Try the web app" button does nothing and is unstyled.
- [ ] We're missing the text on the top bar that says "Remove the code in lines XXX to hide this notification"
- [ ] For some reason I can't see any "Recent Articles" or any blog posts under /blog but I'm not sure if that's your code change or my Space set up.

File: pages/blog/index.vue [hunk 1]
```diff
@@ -1,35 +1,36 @@
...
   <div>
-    <Navigation></Navigation>
-    <div class="callout large">
-      <div class="row column text-center">
-        <h1>Blog</h1>
+    <!-- Blog header -->
+    <header class="blog header">
+      <div class="foreground">
+        <div class="page-bar wrapper">
+          <h1><a href="/">John Doe</a></h1>
```
Comment (joaoramos) [inline pages/blog/index.vue]: This seems pretty bad: both the link back to the home page, and the hard copy.

**Comment by joaoramos** [comment]
👇 I get this a lot when jumping from home page to a blog post. It's as if everything renders but the CSS applied to the header, which is somehow keeping the weird absolute positioning of the picture of the home page and applying it to the blog post picture.

custom app with contentful

**Comment by stefanjudis** [comment]
☝️ @joaoramos This CSS problem should be fixed. :)

File: layouts/default.vue [hunk 2]
```diff
@@ -1,11 +1,117 @@
 <template>
   <div class="container">
+    <Greeting></Greeting>
```
Comment (axe312ger) [inline layouts/default.vue]: I would say 'YES'

Especially since we got now the `WHATS-NEXT.md` :)

Comment (joaoramos) [inline layouts/default.vue]: The problem is that we need to tell people to remove not only line `3` on that file, but also the preceding comment line 😓

Comment (axe312ger) [inline layouts/default.vue]: > Delete lines 3-5 in `layouts/default.vue` to remove the welcome message

**Pull request merged**